credoai/assessment/assessments.py

```diff
@@ -2,12 +2,12 @@
 Module containing all CredoAssessmsents
 """
 
-from credoai.assessment.assessment import CredoAssessment, AssessmentRequirements
+from credoai.assessment.credo_assessment import CredoAssessment, AssessmentRequirements
 from credoai.data.utils import get_data_path
 import credoai.modules as mod
 import sys, inspect
 
-class FairnessAssessment(CredoAssessment):
+class FairnessBaseAssessment(CredoAssessment):
```

Comment:
Not quite sure why the demos folder seem to be missing on this branch, but the `quickstart.ipynb` has the following import which seem to be no longer work with this branch:

`from credoai.assessment.assessments import FairnessAssessment `

Reply:
That's been changed in this branch.